README.md

```diff
@@ -70,6 +68,18 @@ Example usage:
 docker run --rm --runtime nvidia {cuda-container-image-name}
 ```
 
+### Ubuntu Core 22
+
+The required nvidia libraries are available in the nvidia-core22 snap.
+
+This requires connection of the graphics-core22 content interface provided by the nvidia-core22 snap, which should be automatically connected once installed.
+
+### Ubuntu Classic
```

Comment:
The "classic Ubuntu" is an unofficial way of referring to:
```suggestion
### Ubuntu Server / Desktop
```

Reply:
Thank you - fixed